RAMP/unpacker.py

```diff
@@ -70,7 +81,7 @@ def _validate_zip_file(zip_file, max_bundle_size_mb):
     # Check zip content size.
     for zip_info in infolist:
         # Size of the compressed/uncompressed data.
-        if zip_info.file_size > file_size_limit_bytes:
+        if not file.filename.startswith("/deps") and zip_info.file_size > file_size_limit_bytes:
```

Comment:
so before your change
size(.so) was ~= size(.zip)

now after your change this is not the case, and this condition should be adjusted to account for the entire zip

Reply:
my requirement is to check only the .so which goes to the ccs. and not the dep dir

Reply:
then maybe better to 
`if file.filename. endswith(".so") and zip_info.file_size > file_size_limit_bytes:`
?